src/app/auth/components/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,284 @@
+  const [buckupKeyContent, setBackupKeyContent] = useState<string>('');
+  const [showPasswordIndicator, setShowPasswordIndicator] = useState(false);
+  const [isValidPassword, setIsValidPassword] = useState(false);
+  const [isEqualPassword, setIsEqualPassword] = useState(false);
+  const [isEmailSent, setIsEmailSent] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [countDown, setCountDown] = useState<number>(10);
+
+  const [passwordState, setPasswordState] = useState<{
+    tag: 'error' | 'warning' | 'success';
+    label: string;
+  } | null>(null);
+
+  useEffect(() => {
+    if (password.length > 0) onChangeHandler(password);
+    confirmPassword && confirmPassword != password ? setIsEqualPassword(false) : setIsEqualPassword(true);
+  }, [password]);
+
+  useEffect(() => {
+    const confirmPasswordLength = confirmPassword.length;
+    const firstLettersPassord = password.substring(0, confirmPasswordLength);
+    confirmPassword && confirmPassword != firstLettersPassord ? setIsEqualPassword(false) : setIsEqualPassword(true);
+  }, [confirmPassword]);
+
+  useEffect(() => {
+    if (countDown > 0 && isEmailSent) {
+      const timer = setInterval(() => {
+        setCountDown(countDown - 1);
+      }, 1000);
+      return () => clearInterval(timer);
+    }
+
+    countDown === 0 && window.location.assign(`${window.location.origin}/login`);
+  }, [isEmailSent, countDown]);
+
+  const uploadBackupKey = () => {
+    backupKeyInputRef.current?.click();
+  };
+
+  const onUploadBackupKeyInputChanged = async (e) => {
+    const file = e.target.files[0];
+    const backupKey = await file.text();
+    const isValidBackupKey = validateMnemonic(backupKey);
+
+    isValidBackupKey
+      ? setBackupKeyContent(backupKey)
+      : notificationsService.show({
+          text: translate('auth.recoverAccount.changePassword.backupKeyError'),
+          type: ToastType.Error,
+        });
+  };
+
+  function onChangeHandler(input: string) {
+    setIsValidPassword(false);
+    if (input.length > MAX_PASSWORD_LENGTH) {
+      setPasswordState({ tag: 'error', label: translate('modals.changePasswordModal.errors.longPassword') });
+      return;
+    }
+
+    const result = testPasswordStrength(input, '');
+
+    setIsValidPassword(result.valid);
+
+    if (!result.valid) {
+      setPasswordState({
+        tag: 'error',
+        label:
+          result.reason === 'NOT_COMPLEX_ENOUGH'
+            ? translate('auth.recoverAccount.changePassword.notPasswordStrength')
+            : translate('auth.recoverAccount.changePassword.shortPasswordStrength'),
+      });
+    } else if (result.strength === 'medium') {
+      setPasswordState({
+        tag: 'warning',
+        label: translate('auth.recoverAccount.changePassword.warningPasswordStrength'),
+      });
+    } else {
+      setPasswordState({
+        tag: 'success',
+        label: translate('auth.recoverAccount.changePassword.successPasswordStrength'),
+      });
+    }
```

Comment:
You are right, this always accompanies the PasswordStrengthIndicator component, we would have to make a refactor to make it part of the component but I think it is beyond the scope of the task as it involves other parts of the code.
I think we should create a task for this.

Reply:
OK, if it doesn't take too long I would do it in this task. But as you prefer, if it is going to be done in another task, please create the task and assign it so that we don't take too long to solve this technical debt.

Reply:
I think it is better not to block the release of this feature with this, I have created the task and assigned it to me.
https://inxt.atlassian.net/browse/PB-749